code/modules/mob/living/carbon/human/species/golem.dm

```diff
@@ -718,7 +718,7 @@
 	icon = 'icons/obj/items.dmi'
 	icon_state = "pile_bandages"
 	resistance_flags = FLAMMABLE
-	var/revive_time = 900
+	var/revive_time = 90 SECONDS
```

Comment:
![image](https://github.com/user-attachments/assets/d8a91fcd-7463-4cb7-8387-3836aa25a2e0)

# This makes no change